### PR TITLE
More desktop tools, selenium grid, docker, blog additions, etc.

### PR DESCRIPTION
**Pull request by daluu**
See the PR diff. Basically adding overlooked items for:

* Selenium Grid enhancements
* Desktop GUI automation via WebDriver API (the tools for this)
* more desktop IDE / element inspector tools
* device farms category (for use with Selenium)
* rename some headings for better clarification
* add a blog entry

If you think some items need vetting or change, do let me know. I think all the added items are worth inclusion.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,23 +108,49 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
...
 - [Selendroid](http://selendroid.io/mobileWeb.html) - Selendroid is based on the Android instrumentation framework.
 - [ios-driver](http://ios-driver.github.io/ios-driver/) - Test any IOS native, hybrid, or mobile web application using Selenium / WebDriver.  
 - [Winium](https://github.com/2gis/Winium) - Automation framework for Windows platforms. It is free. It is opensource. It is Selenium-based. Supports: Windows Desktop (WPF, WinForms); Windows Store or Universal Apps for Windows Phone; Windows Phone Silverlight Apps.
+- [WebDriverAgent](https://github.com/manishPatwari/WebDriverAgent) - a WebDriver server for iOS to remote control devices via WebDriver API.
+
+#### Desktop GUI Automation (non-browser-centric)
+
+- [WinAppDriver](https://github.com/Microsoft/WinAppDriver) - Microsoft's WebDriver implementation for Windows application automation.
+- [Winium](https://github.com/2gis/Winium) - Automation framework for Windows platforms. It is free. It is opensource. It is Selenium-based. Supports: Windows Desktop (WPF, WinForms); Windows Store or Universal Apps for Windows Phone; Windows Phone Silverlight Apps.
```
Comment (christian-bromann) [inline README.md]: This is already listed under `Mobile (browser and apps)`

File: README.md [hunk 2]
```diff
@@ -66,11 +68,13 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [Selenide](https://github.com/codeborne/selenide) - A framework for writing easy-to-read and easy-to-maintain automated tests using a Fluent API. Selenide has a magic trick that resolves most problems with Ajax and timeouts.
 - [Galen Framework](http://galenframework.com/) - Automated testing of look and feel for your responsive websites.
 - [Serenity](http://www.thucydides.info/) - It is an open source library for writing better quality automated acceptance tests faster. (Formerly Thucydides).
+- [SeLion](https://github.com/paypal/SeLion) - A framework for running Selenium tests with additional features beyond the basic Selenium Grid functionality.
```
Comment (christian-bromann) [inline README.md]: You already defined this under Selenium Grid

**Review by christian-bromann — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (daluu) [reply README.md]: Yes, but Winium does much more than than just mobile if you read the description. Could change the description for it under each section.

Or let me guess, no duplicate listings whatsoever? If that's the case, be interesting to decide where an item best fits when it spans multiple categories.

File: README.md (hunk 2 above)
Comment (daluu) [reply README.md]: Again, let me guess, no duplicate listings whatsoever? If that's the case, be interesting to decide where an item best fits when it spans multiple categories.

File: README.md (hunk 1 above)
Comment (christian-bromann) [reply README.md]: > Or let me guess, no duplicate listings whatsoever?

Yes, feel free to put it into what category fits best in your opinion

File: README.md (hunk 2 above)
Comment (christian-bromann) [reply README.md]: same here

**Review by christian-bromann — APPROVED** [review]
Awesome, great additions 👍

**Pull request merged**